modules/wallet/money.go

```diff
@@ -0,0 +1,46 @@
+package wallet
+
+import (
+	"github.com/NebulousLabs/Sia/types"
+)
+
+// ConfirmedBalance returns the balance of the wallet according to all of the
+// confirmed transactions.
+func (w *Wallet) ConfirmedBalance() (siacoinBalance types.Currency, siafundBalance types.Currency, siafundClaimBalance types.Currency) {
+	lockID := w.mu.Lock()
+	defer w.mu.Unlock(lockID)
+
+	for _, sco := range w.siacoinOutputs {
+		siacoinBalance = siacoinBalance.Add(sco.Value)
+	}
```

Comment:
it occurs to me now that we could actually write a sum function. It would look like:
```go
total := types.CurrencySum(w.siacoinOutputs, "Value")
```
You would need to use reflection though, so it'd be error-prone and slow. Fun to think about though.